frontend/src/components/core/Block/Block.scss

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2018-2019 Streamlit Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import "src/assets/css/variables";
+
+.reportview-container {
+  // Reset the progress bar to 0 without animating on report re-run.
+  .progress > .progress-bar {
+    -moz-transition: none;
+    -webkit-transition: none;
+    -o-transition: color 0 ease-in;
+    transition: none;
```

Comment:
This looks a bit too harsh.

What I was thinking was we'd have a little bit of logic to determine whether the transition should be set to `none` or something else.

Something like:
- If we're going back to `0`, set transition to `none`. This makes the scrollbar return to 0 quickly.
- Otherwise, if we're the time between the previous update and this one was < 50ms, set transition to `none`. This makes the scrollbar stop acting weird when updates come really quickly.
- Finally, if none of those are true, set transition to `width 100ms linear`. I played with some values locally and `100ms` looks smooth without feeling slow, and `linear` interpolation makes the progress bar smoother when moving quickly.

Reply:
Let me know if you want to try it and discuss after scrum today